viewing_party/party.py

```diff
@@ -1,23 +1,181 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    movie_info = {}
+    if not title or not genre or not rating:
+        return None
+
+    # if title and genre and rating:
+
+    movie_info["title"] = title
+    movie_info["genre"] = genre
+    movie_info["rating"] = rating
+
+    return movie_info
+
+result = create_movie("", "Horror", "5")
+print(result)
+
+def add_to_watched(user_data, movie):
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data["watchlist"].append(movie)
+    return user_data
+
+def watch_movie(user_data, title):
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+            user_data["watchlist"].remove(movie)
+            user_data["watched"].append(movie)
+
+    return user_data
+
 
-# -----------------------------------------
 # ------------- WAVE 2 --------------------
-# -----------------------------------------
+def get_watched_avg_rating(user_data):
+    ratings_list = []
+    if not user_data["watched"]:
+        return 0.0
+    for movie in user_data["watched"]:
+        ratings_list.append(movie["rating"])

```

Comment:
I see that y'all have made a list of all the ratings and then you do the summation on line 44 using the built in `sum()` function! I'm usually all for separating out each step, but in this case, it does seem a little unnecessary. We're going to have to write this loop either way, so it might be a good idea to just use the loop for our summation.